packages/office-ui-fabric-react/src/components/Modal/Modal.base.tsx

```diff
@@ -150,6 +156,17 @@ export class ModalBase extends BaseComponent<IModalProps, IDialogState> implemen
     }
   }
 
+  // Allow the user to scroll within the modal but not on the body
+  private _allowScrollOnModal(elt: HTMLDivElement | null): void {
+    if (elt) {
+      this._scrollableContent = elt;
+      allowScrollOnElement(this._scrollableContent, this._events);
+    } else {
+      this._events.off(this._scrollableContent);
+      this._scrollableContent = null;
```

Comment:
you can move this line down below the if to reduce repetition. (assign scrollableContent = elt)